src/features/home/sections/QASection/QASection.module.scss

```diff
@@ -0,0 +1,22 @@
+@import '@/theme/variables';
+@import '@/theme/mixins';
+
+.containerQA {
```

Comment:
use scss nesting

Reply:
Added the requested changes. Please take a look at Accordion.module.scss file to confirm if I implemented the nesting correctly as you suggested. But I think I get the idea.